frontend/views/containers/forms/FormLogin.vue

```diff
@@ -50,7 +51,17 @@ export default {
     validationsDebouncedMixins
   ],
   components: {
-    FormPassword
+    FormPassword,
+    FeedbackBanner
+  },
+  data () {
+    return {
+      form: {
+        name: null,
+        password: null,
+        submitFeedback: {}
```

Comment:
This shouldn't be in the `form`, since it's not a form field value. `ephemeral` is probably a better place for it

Reply:
Note: can ignore above comment if [this other approach](https://github.com/okTurtles/group-income-simple/pull/779/files#r348781980) works, since we won't need to have `submitFeedback` as a data variable on the parent